test_programs/plonky2_trace/2_function_calls/expected_trace.json

```diff
@@ -3,20 +3,18 @@
   { "Function": { "path_id": 0, "line": -1, "name": "<toplevel>" } },
   { "Call": { "function_id": 0, "args": [] } },
   { "Type": { "kind": 30, "lang_type": "None", "specific_info": { "kind": "None" } } },
-  { "Path": "?" },
-  { "Step": { "path_id": 1, "line": -1 } },
   { "Path": "/home/stan/code/repos/noir/test_programs/plonky2_trace/2_function_calls/src/main.nr" },
```

Comment:
Maybe think of a way to avoid hardcoding full paths in test files.

I ran `cargo test trace` locally and it failed with `assertion left == right failed`

![image](https://github.com/blocksense-network/noir/assets/51385502/1a4fc995-4596-4b81-9e8d-7f13a99be972)